server/status.go

```diff
@@ -52,10 +51,13 @@ const (
 	// statusLocalStacksKey exposes stack traces of running goroutines.
 	statusLocalStacksKey = statusLocalKeyPrefix + "stacks"
 
-	// statusNodesKeyPrefix exposes status for each of the nodes the cluster.
-	// GETing statusNodesKeyPrefix will list all nodes.
-	// Individual node status can be queried at statusNodesKeyPrefix/NodeID.
-	statusNodesKeyPrefix = statusKeyPrefix + "nodes/"
+	// statusNodesSingleKeyPrefix and statusNodeAllKeyPrefix exposes status for
+	// each of the nodes the cluster.
+	// statusNodesKeyPrefix/nodes -> lists all nodes
+	// statusNodesKeyPrefix/nodes/ -> lists all nodes
+	// statusNodesKeyPrefix/nodes/{NodeID} -> shows only the status for that single node
+	statusNodesSingleKeyPrefix = statusKeyPrefix + "nodes/"
+	statusNodesAllKeyPrefix    = statusKeyPrefix + "nodes"
```

Comment:
As per our offline discussion, I'm going to remove the extra prefix.  I'm going to add https://github.com/julienschmidt/httprouter as a new router, which will make routing more efficient, robust and faster.